src/System.Runtime.Extensions/tests/System/EnvironmentTests.cs

```diff
@@ -217,6 +223,8 @@ public void GetFolderPath_Unix_PersonalIsHomeAndUserProfile()
         [InlineData(Environment.SpecialFolder.LocalApplicationData, Environment.SpecialFolderOption.DoNotVerify)]
         [InlineData(Environment.SpecialFolder.Desktop, Environment.SpecialFolderOption.DoNotVerify)]
         [InlineData(Environment.SpecialFolder.DesktopDirectory, Environment.SpecialFolderOption.DoNotVerify)]
+        // Not set on Unix (amongst others)
+        //[InlineData(Environment.SpecialFolder.System, Environment.SpecialFolderOption.DoNotVerify)]
```

Comment:
Because I initially added that test thinking it was missed. But I realized the code that sets this is just OSX/Windows. Figured I'd record that fact just as you did below in another test.